src/flight_booking_agent/services/amadeus_client.py

```
@@ -0,0 +1,66 @@
+
+load_dotenv()
+
+class AmadeusClient:
+    def __init__(self):
+        try:
+            self.client = Client(
+                client_id=os.getenv("AMADEUS_CLIENT_ID"),
+                client_secret=os.getenv("AMADEUS_CLIENT_SECRET"),
+                hostname=os.getenv("AMADEUS_HOSTNAME", "test") # Mặc định là môi trường test
+            )
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)